docs/application/native/guides/connectivity/softap.md

```diff
@@ -0,0 +1,342 @@
+# SoftAP
+
+SoftAP means software access point for local area connections without internet access.
+
+The main features of the SoftAP API includes:
+
+- Enable or disable SoftAP
+
+   You can turn SoftAP on or off.
+
+
+- Manage SoftAP
+
+   You can manage the SoftAP configuration settings such as Service Set Identifier (SSID) and passphrase.
+   The following table lists the attributes of SoftAP:
+
+   **Table: The attributes of SoftAP**
+    <table>
+     <thead>
+      <tr>
+       <th>Attributes</th>
+       <th>Description</th>
+      </tr>
+     </thead>
+     <tbody>
+      <tr>
+       <td>SSID</td>
+       <td>Service Set Identifier (SSID)</td>
+      </tr>
+      <tr>
+       <td>Security type</td>
+       <td>None, WPA2_PSK, and WPS</td>
+      </tr>
+      <tr>
+       <td>Passphrase</td>
+       <td>Passphrase for wireless access</td>
+      </tr>
+      <tr>
+       <td>Visibility</td>
+       <td>Visibility of SSID</td>
+      </tr>
+      <tr>
+       <td>IP address</td>
+       <td>Local IP address</td>
+      </tr>
+      <tr>
+       <td>Mode</td>
+       <td>Wireless mode (B, G, A, and AD)</td>
+      </tr>
+      <tr>
+       <td>Channel</td>
+       <td>Channel</td>
+      </tr>
+     </tbody>
+    </table>
+
+- SoftAP client
+
+    You can get the information about connected client such as name, IP address, MAC address, and connection time.
+
+
+## Prerequisites
+
+1. To use the SoftAP API (in [mobile](../../api/mobile/latest/group__CAPI_NETWORK_SOFTAP_MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI_NETWORK_SOFTAP_MODULE.html) applications), the application must request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+   ```
+   <privileges>
+     <!--Needed for configuring SoftAP settings-->
+     <privilege>http://tizen.org/privilege/softap</privilege>
+     <!--Needed for enabling and disabling SoftAP-->
+     <privilege>http://tizen.org/privilege/softap.admin</privilege>
+   </privileges>
+   ```
+2. To use the functions and data types of the SoftAP API, include the `<softap.h>` header file in your application:
+
+   ```
+   #include <softap.h>
+   ```
+
+
+## Creating SoftAP handle
```

Comment:
@SeonahMoon 
GLOBAL COMMENT:
It is recommended to use title case for headings in Tizen docs. Please update the headings.
For example, "Creating SoftAP Handle".